server/handlers/meshery_pattern_handler.go

```diff
@@ -128,9 +128,9 @@ func (h *Handler) handlePatternPOST(
 	}()
 
 	var err error
-	action := models.Create
+	action := models.Import
```

Comment:
We wanted to replace create by import ? The title is misleading

Reply:
Good call. I believe that we're trying to make "create" synonymous with "import", preferring the latter term in the case of a user uploading a local file or importing remote file via URL.

Reply:
@MUzairS15 do we continue to have a create event? What event are we sending when a user starts a design from scratch?